tests/Microsoft.DotNet.UpgradeAssistant.TestHelpers/FileHelpers.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.UpgradeAssistant
+{
+    public static class FileHelpers
+    {
+        private static readonly string[] DirsToIgnore = new[] { "bin", "obj" };
+
+        public static async Task CopyDirectoryAsync(string sourceDir, string destinationDir)
```

Comment:
Could we have this as an object that tracks the copied files and exposes an IDisposable to clean them up?

Reply:
Good idea. I'll do that.